DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -288,7 +288,7 @@
   </class>
 
   <class name="pat::PackedCandidate" ClassVersion="28">
-    <version ClassVersion="28" checksum="1956349111"/>  
+    <version ClassVersion="28" checksum="2646473453"/>
```

Comment:
- this has to be a new version number
- this part will have to be updated after #18092 is merged